### ISSUE TITLE
White screen when launching the app

### ISSUE THREAD

**Issue by TudorAndrei** [StarlaneStudios/Surrealist#85]
**Describe the bug**
When launching the app, it opens a white screen.

**To Reproduce**
Launch the app. I launched it from rofi and from the cli.

**Expected behavior**
the app should start normally

**Environment (you can view the current version under Settings):**
 - Operating System: Ubuntu 22.04.2 LTS x86_64, Kernel: 5.19.0-46-generic 
 - Surrealist Version : 1.7.5, 1.7.3
 - SurrealDB Version:  1.0.0-beta.9+20230402.5eafebd for linux on x86_64

**Comment by gander** [comment]
![image](https://github.com/StarlaneStudios/Surrealist/assets/421597/9e846fbf-170f-47e1-99d3-024f841b25b3)

**Comment by macjuul** [comment]
The console tab in the developer tools may provide answers to what causes this, you should be able to open it using `Ctrl + Shift + I`

**Comment by gander** [comment]
Too late. I uninstalled the deb and used AppImage. Information on how to open the console should be somewhere at the top. It's strange, because I also had problems with AppImage for a while, and then it calmed down.

**Comment by vinodh013** [comment]
Hi @macjuul  I am also facing the same issue and this is what I found after opening the dev tool

![Screenshot 2023-09-05 234838](https://github.com/StarlaneStudios/Surrealist/assets/65361302/376424c5-29ce-4a7d-aef4-05fd459a1dec)

**Comment by macjuul** [comment]
@vinodh013 this appears to be caused by a malformed or corrupted config file. Try deleting `.surrealist.json` in your home directory and see if that fixes it

**Comment by vinodh013** [comment]
Thank you @macjuul it's working perfectly after deleting `.surrealist.json`

**Comment by nerdo** [comment]
I have had a similar issue for a while. It started with an older version of surrealist and I just gave up on it.

I came back to try it again now that surrealdb 1.0 is released, downloaded the latest version of surrealist, and same thing, white screen on launch - app is unresponsive.

I don't have a `.surrealist.json` file in my home directory, so I'm not sure what's going on or how to fix it.

Oh, this is happening to me on macos by the way, not in Linux
